ext/standard/basic_functions.stub.php

```diff
@@ -57,11 +57,11 @@ function stream_wrapper_restore(string $protocol): bool {}
 
 /* array.c */
 
-function array_push(array &$stack, mixed ...$args): int {}
+function array_push(array &$stack, mixed ...$value): int {}
```

Comment:
Most (if not all) other varargs are plural in php src. So s/value/values/

Reply:
Wouldn't that cause an issue if used singly?

```php
array_push(value=5, stack=$arr);
// vs
array_push(values=5, stack=$arr);
```

Reply:
Apparently with varargs "You cannot target the variadic parameter via named args".

Also, I think the names for varargs should match what they would be for userland functions, where a vararg would definitely be named as the plural.